java/launcher/src/main/java/com/lightstep/launcher/server/ExampleServer.java

```diff
@@ -0,0 +1,62 @@
+package com.lightstep.launcher.server;
+
+import com.lightstep.opentelemetry.launcher.OpenTelemetryConfiguration;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Tracer;
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.ContextHandlerCollection;
+
+public class ExampleServer {
+
+  public static void main(String[] args) throws Exception {
+    final String spanEndpoint = System.getenv("OTEL_EXPORTER_OTLP_SPAN_ENDPOINT");
+    final String lsToken = System.getenv("LS_ACCESS_TOKEN");
+    final String serviceName = System.getenv("LS_SERVICE_NAME");
+    final String serviceVersion = System.getenv("LS_SERVICE_VERSION");
+
+    OtlpGrpcSpanExporter exporter = OpenTelemetryConfiguration.newBuilder()
```

Comment:
same question as above re: why you have to set these?

Reply:
removed